### ISSUE TITLE
Problems in (high profile) GLM Hierarchical notebook

### ISSUE THREAD

**Issue by cluhmann** [pymc-devs/pymc-examples#421]
**Notebook title**:GLM: Hierarchical Linear Regression
**Notebook url**: [source link](https://github.com/pymc-devs/pymc-examples/blob/main/examples/generalized_linear_models/GLM-hierarchical.ipynb), [website link](https://www.pymc.io/projects/examples/en/latest/generalized_linear_models/GLM-hierarchical.html)

## Issue description
There are many, many divergences in the sample of the main, hierarchical model. There also appear to be images missing under "The Models" section.  Not sure what these are supposed to be.

## Proposed solution
I suspect the sampling could be fixed by re-running and/or setting the random seed for stability.  Not sure what the images were (at one time).

**Comment by OriolAbril** [comment]
Maybe we can remove/deprecate it in favour of https://www.pymc.io/projects/examples/en/latest/case_studies/multilevel_modeling.html? Or probably better combine/restructure the two to link between them to make them complementary and avoid duplication.

cc @fonnesbeck who I know was thinking about improving the multilevel modeling primer one.

**Comment by fonnesbeck** [comment]
Yes, there are a few things about the current version of the multilevel modeling primer I don't like (notably the expression of floor and basement as being independent). There is significant overlap in content with this notebook, so I'd be in favor of somehow merging them. If I'm not mistaken, this notebook originally used the now-extinct `glm` submodule, which made it stand out a little more. As far as the sampling issues go, I'd bet they are related to the centered random effect parameterization.

**Comment by fonnesbeck** [comment]
UPDATE: Will be aiming to complete this in the next few days.
